public/app/features/explore/QueryField.tsx

```diff
@@ -468,6 +468,14 @@ export class QueryField extends React.PureComponent<QueryFieldProps, QueryFieldS
     );
   };
 
+  handlePaste = (event: ClipboardEvent, change: Editor) => {
```

Comment:
Nice catch!

Do we know if this bug is confirmed by slate?

Reply:
Later versions of Slate handles this automagically but introduce other errors/breaking changes.